build/templates/reportTemplate.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <link href="style.css" rel="stylesheet">
+      <link href="bootstrap-theme.min.css" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
+      <title>report generated on {{  datetime }}</title>
+   </head>
+   <body>
+      <header class="header">
+         <nav class="navbar navbar-expand-lg navbar-dark">
+            <!-- Navbar content -->
+            <div class="container">
+               <a class="navbar-brand" href="https://www2.gov.bc.ca">
+               <img class="img-fluid d-none d-md-block" src="https://developer.gov.bc.ca/static/BCID_H_rgb_rev-20eebe74aef7d92e02732a18b6aa6bbb.svg" width="177" height="44" alt="B.C. Government Logo">
+               </a>
+               <div class="navbar-brand">
```

Comment:
I just saw in your presentation, that the heading is aligned right. Just an FYI, but following BC Gov standards this should be left aligned.